metrics/src/main/java/org/wildfly/extension/metrics/WildFlyMetric.java

```diff
@@ -79,7 +78,7 @@ private ModelNode readAttributeValue(PathAddress address, String attributeName)
             } else{
                 LOGGER.unableToReadAttribute(attributeName, address, error);
             }
-            return new ModelNode(ModelType.UNDEFINED);
+            return new ModelNode();
```

Comment:
I think, we would better having a constant (`static final ModelNode UNDEFINED = new ModelNode()`) to avoid having to create many instances of the same object here

Reply:
Ah, ok. That's what you meant by using a constant. :P One moment...